packages/components/src/font-size-picker/test/index.tsx

```diff
@@ -525,6 +525,39 @@ describe( 'FontSizePicker', () => {
 			expect( screen.getByLabelText( 'Custom' ) ).toBeInTheDocument();
 		} );
 
+		it( 'hides custom input when disableCustomFontSizes is set to `true` with a custom font size', () => {
+			const { rerender } = render(
+				<FontSizePicker fontSizes={ fontSizes } value="3px" />
+			);
+			expect( screen.getByLabelText( 'Custom' ) ).toBeInTheDocument();
```

Comment:
I usually prefer `.toBeVisible()` for existing visible elements, and `.not.toBeInTheDocument()` for asserting non-existing ones, because `.toBeInTheDocument()` doesn't guarantee that the element is actually visible.

Reply:
Changed in https://github.com/WordPress/gutenberg/pull/63553/commits/ae4a0d233d726ca9066e18712d7bd2e9545f8a6e for all occurrences in the file